### ISSUE TITLE
stubtest: Abstract properties are not detected

### ISSUE THREAD

**Issue by sobolevn** [python/mypy#13646]
Code:

```python
from abc import abstractmethod

class Some:
    @property
    @abstractmethod
    def x(self) -> int:
         return 1
```

Stub:

```python
class Some:
    @property
    def x(self) -> int: ...
```

No errors are shown.
It should raise an error about abstract state mismatch.
Realted https://github.com/python/typeshed/pull/8726

**Comment by AlexWaygood** [comment]
I thought you already fixed this in https://github.com/python/mypy/pull/13323#issuecomment-1205336649? It just hasn't been released yet?

**Comment by sobolevn** [comment]
Sorry, I did not mention: I am using `master` version (`0f17aff06ac1c05c442ba989e23655a2c6adbfbf`) of course. We even have a test for it here: https://github.com/python/mypy/blob/0f17aff06ac1c05c442ba989e23655a2c6adbfbf/mypy/test/teststubtest.py#L1391-L1403

But, for some reason the example above is not reported.

**Comment by sobolevn** [comment]
Oh, this is why: 

```python
             class AP1: 
                 def some(self) -> int: ... 
```

We miss `@property` decorator there.

**Comment by AlexWaygood** [comment]
Got it.
